.gitignore

```diff
@@ -42,19 +32,18 @@ RUNNING_PID
 
 local.sbt
 
-node_modules
+dev/node_modules
+tools/amp-validation/node_modules
+static/src/stylesheets/pasteup/node_modules/
+/node_modules
+
```

Comment:
This was added to prevent the `deploys radiator` being ignored thus validation errors.